e2e/email-sign-up.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { test, expect, type Page } from '@playwright/test';
+import translations from '../client/i18n/locales/english/translations.json';
+
+let page: Page;
```

Comment:
```suggestion
import { test, expect } from '@playwright/test';
```

Remove the unused translations import and global `page` variable.